docs/07-test-timeouts.md

```diff
@@ -13,3 +13,12 @@ npx ava --timeout=10s # 10 seconds
 npx ava --timeout=2m # 2 minutes
 npx ava --timeout=100 # 100 milliseconds
 ```
+
+Timeouts can also be set individually for each test. These timeouts are reset each time an assertion is made.
+
+```javascript
+test('foo', t => {
+	t.timeout(100); # 100 milliseconds
+	...
+});
+```
```

Comment:
Could you duplicate this section in https://github.com/avajs/ava/blob/master/docs/02-execution-context.md?